src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,73 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
```suggestion
    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names